notebooks/COS/ViewData/ViewData.ipynb

```diff
@@ -0,0 +1,1452 @@
+    "\n",
+    "- For an in-depth manual to working with COS data and a discussion of caveats and user tips, see the [COS Data Handbook](https://hst-docs.stsci.edu/display/COSDHB/).\n",
+    "- For a detailed overview of the COS instrument, see the [COS Instrument Handbook](https://hst-docs.stsci.edu/display/COSIHB/).\n",
+    "\n",
+    "#### Notes for those new to Python/Jupyter/Coding:\n",
+    "\n",
+    "- You will frequently see exclamation points (**\\!**) or dollar signs (**\\$**) at the beginning of a line of code we are telling you to run. These are not part of the actual commands. The exclamation points tell a jupyter notebook to pass the following line to the command line, and the dollar sign merely indicates the start of a terminal prompt. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## We will import the following packages:\n",
+    "\n",
+    "- `numpy` to handle array functions (version $\\ge$ 1.17)\n",
+    "  - We also insert a line telling numpy to ignore a certain type of warning about divide-by-zeros. This is not always a good idea, so seriously consider it before copying it into your code!\n",
+    "- `astropy.io fits` for accessing FITS files\n",
+    "- `astropy.table Table` for creating tidy tables of the data\n",
+    "- `astropy.units` and `astropy.visualization.quantity_support` for dealing with units\n",
+    "- `matplotlib.pyplot` for plotting data\n",
+    "- `astroquery.mast Mast and Observations` for finding and downloading data from the [MAST](https://mast.stsci.edu/portal/Mashup/Clients/Mast/Portal.html) archive\n",
+    "\n",
+    "*Later on, we will import some functions from a local file*, for which we have imported the `sys` package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This line is just to make matplotlib look good in a notebook\n",
+    "%matplotlib inline\n",
+    "# Manipulating arrays\n",
+    "import numpy as np\n",
+    "# Our arrays will encounter divide by 0 warnings, which is OK, but we want to suppress the warning text with the next code line.\n",
+    "   # SO DO NOT COPY THIS NEXT LINE INTO YOUR CODE WITHOUT CAREFUL CONSIDERATION!\n",
+    "np.warnings.filterwarnings('ignore')\n",
+    "# Reading in data\n",
+    "from astropy.io import fits\n",
+    "from astropy.table import Table\n",
+    "# Dealing with units and plotting them\n",
+    "from astropy import units as u\n",
+    "from astropy.visualization import quantity_support\n",
+    "quantity_support()\n",
+    "# Plotting\n",
+    "import matplotlib.pyplot as plt\n",
+    "# Downloading data from archive\n",
+    "from astroquery.mast import Mast\n",
+    "from astroquery.mast import Observations\n",
+    "from astroquery.mast import Catalogs\n",
+    "\n",
```

Comment:
I suggest you make more use of linebreaks in this cell - right now the "wall of imports" is a bit hard to read.  Consider something like what's in https://learn.astropy.org/rst-tutorials/Coordinates-Intro.html?highlight=coordinates#imports